### ISSUE TITLE
Github PRs being unexpectedly filtered out

### ISSUE THREAD

**Issue by adamhigerd** [travis-ci/travis-ci#6286]
Our repository is configured [as described in this StackOverflow thread](http://stackoverflow.com/questions/31882306/how-to-configure-travis-ci-to-build-pull-requests-merges-to-master-w-o-redunda) and it was working fine for about five months until yesterday's Travis update. Now, pushes to the master branch are still tested, but PRs aren't.

After some digging, I've established that the webhooks are working, but Travis is skipping them with a `branch not included or excluded` message visible on the "Requests" page.

As a workaround, I've removed the `branches: only: master` clause from my .travis.yml file and turned off building PRs in my Travis configuration, so my pushes are at least getting tested, but they're not being treated as PRs by my scripts.

**Comment by BanzaiMan** [comment]
Do you have a build log that shows the problem you are describing here?

**Comment by adamhigerd** [comment]
No, I don't have a build log, because my issue is that it's skipping the build entirely. I can, at best, take a screenshot of the Requests page that shows a bunch of push notifications being marked as `branch not included or excluded`.

(In the case of the workaround, not being treated as PRs is working as intended; I have a build log there but it's irrelevant.)

**Comment by BanzaiMan** [comment]
Oh, right. Please indicate the affected repository, the commit that caused this problem and the corresponding `.travis.yml`. Thanks.

**Comment by adamhigerd** [comment]
I can't isolate any specific commit that caused the problem. Nothing changed on our end; the builds just stopped happening out of the blue on multiple branches.

The repository is `moovweb/moovcheckout-catalyst`. The corresponding [.travis.yml](https://github.com/travis-ci/travis-ci/files/355722/travis.yml.txt) is attached.

**Comment by BanzaiMan** [comment]
@adamhigerd Someone appears to have disabled building PRs. Please go to https://travis-ci.com/moovweb/moovcheckout-catalyst/settings and enable it.

**Comment by adamhigerd** [comment]
I know. **_I_** disabled building PRs. I said that in my issue description -- it's part of a workaround to get SOME level of CI testing running because the configuration we'd been running successfully for five months suddenly stopped working.

**Comment by BanzaiMan** [comment]
Ah. Right. Sorry about the confusion there. We'll investigate further. Thanks.

**Comment by BanzaiMan** [comment]
Sorry about the delay. Could you email us at support@travis-ci.com?
